README.md

```diff
@@ -1,19 +1,289 @@
+
+---
+
+## Jobs
+
+### run
+
+A single, complete job to run Cypress tests in your application
+
+## Basic Setup
+
+A typical project can have:
+
+```yaml
 version: 2.1
 orbs:
-    # import Cypress orb by specifying an exact version x.y.z
-    # or the latest version 3.x.x using "@3" syntax
-    cypress: cypress-io/cypress@3
+  # "cypress-io/cypress@3" installs the latest published
+  # version "s.x.y" of the orb. We recommend you then use
+  # the strict explicit version "cypress-io/cypress@3.x.y"
+  # to lock the version and prevent unexpected CI changes
+  cypress: cypress-io/cypress@3
 workflows:
-    build:
-        jobs:
-            # "cypress" is the name of the imported orb
-            # run is the name of the job defined in Cypress orb
-            - cypress/run
-```
\ No newline at end of file
+  build:
+    jobs:
+      - cypress/run # "run" job comes from "cypress" orb
+```
+
+By using the `run` job definitions that Cypress provides inside the orb, it
+brings simplicity and static checks of parameters to your CircleCI
+configuration.
+
+You can find multiple examples at
+[our official orb page](https://circleci.com/developer/orbs/orb/cypress-io/cypress)
+
+### Arguments
+
+**install-command** **_(String)_**
+
+Overrides the default NPM command (npm ci)
+
+**post-install** **_(String)_**
+
+Additional command(s) to run after running install but before verifying Cypress
+and saving cache.
+
+**cypress-cache-key** **_(String)_**
+
+Cache key used to cache the Cypress binary.
+
+> The default value is `cypress-cache-{{ arch }}-{{ checksum: "package.json" }}`
+
+**cypress-cache-path** **_(String)_**
+
+By, default, this will cache the `~/.cache/Cypress` directory so that the
+Cypress binary is cached. You can override this by providing your own cache path
+
+**node-cache-version** **_(String)_**
+
+Used to change the default node cache version (v1) if you need to clear the
+cache for any reason.
+
+**include-branch-in-node-cache-key** **_(String)_**
+
+If true, the node cache will only apply to jobs within the same branch. Turned
+off by default
+
+**working-directory** **_(String)_**
+
+Directory containing the `package.json` file. Used primarily for those using
+monorepos.
+
+**package-manager** **_(Enum)_**
+
+Select the default node package manager to use. NPM v5+ required.
+
+> Available values include: `npm` (default), `yarn`, or `yarn-berry`
+
+**start-command** **_(String)_**
+
+Command used to start your local dev server for Cypress to test against
+
+**cypress-commandy** **_(String)_**
+
+Command used to run your Cypress tests including any flags you want to use
+
+**parallelism** **_(Integer)_**
+
+Number of Circle machines to use for load balancing, min 1 (requires `parallel`
+and `record` flags in your `cypress-command`). Defaults to 1
+
+**install-browsers** **_(Boolean)_**
+
+Flag used to install additional browsers to run your tests. This is only needed
```

Comment:
We may want to clarify that Cypress uses Electron by default, so if you need other browsers...

Reply:
May also be a good time to clarify this is using https://circleci.com/developer/orbs/orb/circleci/browser-tools so it's clear why edge isn't included, etc